docs/_articles/en/codebuilder/cb-tips.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Useful Tips for Using Code Builder
+lang: en
+---
+
+## Install Extensions from Open VSX Registry
+
+Development experience in Code Builder can be greatly enhanced by installing additional extensions to your workspace. [Open VSX](open-vsx.org) is an Eclipse open-source project and alternative to the Visual Studio Marketplace. Extensions available here can be installed in Code Builder. You can browse and install these extensions from within VS Code. Bring up the Extensions view by clicking on the Extensions icon in the Activity Bar or the **View: Extensions** command (⇧⌘X).
```

Comment:
@sbudhirajadoc - A few suggested edits - alternate version:

Maximize your productivity in Code Builder by installing additional extensions you work with to your Code Builder environment. [Open VSX](open-vsx.org) is an Eclipse open-source project and and public forum of Visual Studio Code extensions. Extensions available here can be installed in Code Builder, provided they are compatible with a web-based environment. You can browse and install these extensions from within Code Builder. Bring up the Extensions view by clicking on the Extensions icon in the Activity Bar or the **View: Extensions** command (⇧⌘X).